### PR TITLE
Weekly update for 2015-07-31.

### PR DESCRIPTION
**Pull request by diagramatics**
Notifying changes:
- A prompt for readers on suggesting content for Upcoming Events and Community Updates.
- Links to articles on Community Updates have the utm_source and utm_medium appended. Discussions are at #124.

Also, if it's not too much, pinging @Trott for nitpicking request. I did rewrite the version announcement section a little bit.

### CODE REVIEW

File: weekly-updates/weekly-update.2015-07-31.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+* Calling `dns.setServers()` while a DNS query is in progress can cause the process to crash on a failed assertion. [#894](https://github.com/nodejs/io.js/issues/894)
+* `url.resolve` may transfer the auth portion of the url when resolving between two full hosts, see [#1435](https://github.com/nodejs/io.js/issues/1435).
+
+### Community Updates
+
+* We need a V8 maintainer for our LTS build! Head on over [to GitHub](https://github.com/nodejs/LTS/issues/28) to see if the requirements match your capabilities.
+* [Milo Mordaunt](https://twitter.com/bananaoomarang) has written about [extending IFTTT with Webtask.io running Node.js code](https://auth0.com/blog/2015/07/28/if-this-then-node-dot-js-extending-ifttt-with-webtask-dot-io/?utm_source=io.js+and+Node.js+News&utm_medium=article) over on [Auth0](https://auth0.com/blog).
+* [Hemanth.HM](https://twitter.com/gnumanth), an author and contributor of more than 200 modules on npm has written about [authoring Node modules](http://h3manth.com/new/blog/2015/authoring-node-modules/?utm_source=io.js+and+Node.js+News&utm_medium=article).
```
Comment (Trott) [inline weekly-updates/weekly-update.2015-07-31.md]: Nit: Add a comma after `on npm`.

Comment (diagramatics) [inline weekly-updates/weekly-update.2015-07-31.md]: Fixed, thanks!

**Comment by c0b41** [comment]
@diagramatics  Status?

**Comment by diagramatics** [comment]
Just waiting for a LGTM from the evangelism team.

**Comment by c0b41** [comment]
@diagramatics thanks ı'm wating

File: weekly-updates/weekly-update.2015-07-31.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+If you have spotted or written something about Node.js and io.js, do come over to our [Evangelism team repo](https://github.com/nodejs/evangelism) and suggest it on the [Issues page](https://github.com/nodejs/evangelism/issues), specifically the Weekly Updates issue.
+
+### Upcoming Events
+
+* [BrazilJS Conf](http://braziljs.com.br/) tickets are on sale, August 21st - 22nd at Shopping Center BarraShoppingSul
+* [NodeConf EU](http://nodeconf.eu/) tickets are on sale, September 6th - 9th at Waterford, Ireland
+* [Node.js Italian Conference](http://nodejsconf.it/) tickets are on sale, October 10th at Desenzano - Brescia, Italy
+* [JSConf CO](http://www.jsconf.co/), October 16th - 17th at Ruta N, Medellin
+
+Have an event about Node.js and io.js coming up? You can put your events here through the [Evangelism team repo](https://github.com/nodejs/evangelism) and announce it in the [Issues page](https://github.com/nodejs/evangelism/issues), specifically the Weekly Updates issue.
```
Comment (yosuke-furukawa) [inline weekly-updates/weekly-update.2015-07-31.md:39]: :+1:

**Pull request merged**